src/example-relay/src/Homepage/locations/LocationsList.js

```diff
@@ -10,10 +10,12 @@ type Props = {|
   +start?: number,
 |};
 
-export default function LocationList({ children, ...rest }: Props): React.Node {
+export default function LocationList({ children, start = 1 }: Props): React.Node {
   return (
-    <ol {...rest} className={styles('ol')}>
-      {children}
+    <ol start={start} className={styles('ol')}>
+      {React.Children.map(children, (child, i) =>
+        React.cloneElement(child, { liClass: styles('li'), dataCount: start + i }),
```

Comment:
Why is the `li` class being styled here (and passed down) rather than near the actual `li` element? 🤔

Reply:
Yeah, with the initial solution I think it made more sense because I had to put the counter-reset on the `ol` and use it also on the li, so it would be less `wtf`, where does this come from. 

I think now with this solution, I can move the styling to the `li` instead

Reply:
🚀